angelman/angelman/forms/angelman_registration_form.py

```diff
@@ -75,6 +102,7 @@ def __init__(self, *args, **kwargs):
     surname = CharField(required=True, max_length=30)
     date_of_birth = DateField(required=True)
     gender = ChoiceField(choices=Patient.SEX_CHOICES, widget=RadioSelect, required=True)
+    diagnosis = ChoiceField(required=True, widget=Select, choices=_diagnosis, initial="")
```

Comment:
Is this supposed to be referring to a fixed (at startup) value? If so, then I think this should've referred to `diagnosis_choices` rather than `_diagnosis`, but that won't work without some adjustments.

If it's not fixed, then I think you can delete `diagnosis_choices = _diagnosis()`, and not handle missing tables in `_diagnosis`

Reply:
Yes, good point.